### ISSUE TITLE
Relatie tussen Tentoonstelling en objecten

### ISSUE THREAD

**Issue by brechtvdv** [Informatievlaanderen/OSLOthema-cultureelErfgoed#15]
Momenteel is het mogelijk om een Tentoonstelling te definiëren via Activiteit waarbij Entiteit.type = "http://vocab.getty.edu/aat/300054766" is. Dit is in lijn met Linked Art.

Weliswaar is het nu niet mogelijk om tentoongestelde objecten te linken met deze activiteit.
Linked Art definieert een set (la:Set) van tentoongestelde objecten en linkt deze set met de activiteit via crm:P16_used_specific_object . Voor het OSLO model zou ik voorstellen om Collectie ipv set hiervoor te gebruiken.

Een oplossing zou dus zijn om een relatie toe te voegen in AP Cultureel Erfgoed Event bij Activiteit: gebruiktSpecifiekObject (crm:P16_used_specific_object) dat verwijst naar een Collectie.

**Comment by dimi-schepers** [comment]
Is Tentoonstelling een Voorstelling en Collectie tegelijkertijd?

**Comment by brechtvdv** [comment]
Nee, een tentoonstelling verwijst naar een set/collectie van tentoongestelde objecten

**Comment by dimi-schepers** [comment]
Voorstel om de associatie "gebruikt specifiek object" ([http://www.cidoc-crm.org/cidoc-crm/P16_used_specific_object](http://www.cidoc-crm.org/cidoc-crm/P16_used_specific_object)) op de klasse Activiteit toe te voegen. CIDOC geeft deze associatie een range van E70 Thing. Deze klasse komt echter niet voor in het AP.

Vraag voor de werkgroep: op welke klasse willen wij de range zetten? Enkele voorstellen en hun nadelen:
- Entiteit (cidoc:E1): is breder dan de range die CIDOC toelaat
- Collectie (dc:Collection): is nu niet gedefinieerd als een subklasse van E70 Thing
- MaterieelDing (cidoc:E18): sluit Informatie Object uit (wat CIDOC niet doet) alsook Collectie

**Comment by brechtvdv** [comment]
Collectie zou voor deze use case genoeg zijn, maar zou het zeker zo breed mogelijk houden.

Misschien is https://data.vlaanderen.be/doc/applicatieprofiel/generiek-basis/#Object (prov:Entity) een optie om te gebruiken?
Conflicteert niet zoals cidoc:E1 volgens mij

**Comment by GeertThijs** [comment]
Goed idee om P16 hiervoor te gebruiken, waarbij een Tentoonstelling een E7 Activity is. We hebben Activiteit in het AP "Cultureel erfgoed Event" overgenomen uit CIDOC en kunnen dit attribuut daar dus toevoegen. Enkel te gebruiken weliswaar voor zaken die een E70 Thing zijn. We hebben deze klasse niet expliciet opgenomen, maar wel de subklassen E28 ConceptualThing (ConceptueelDing) en E18 PhysicalThing (MaterieelDing) en van deze beide klassen subklassen als E22 HumanModeObject, E25 HumanMadeFeature, E78 Collection, E73 InformationObject, F1 Work etc. Merk op dat E78 Collection (GecureerdeCollectie) in dit rijtje staat en we zo een tentoonstelling als Activiteit inderdaad gaan kunnen koppelen aan een (voor hety doel van de tentoonstelling) samengebrachte set objecten.

**Comment by GeertThijs** [comment]
En we gaan uiteraard ook de klasse Tentoonstelling moeten toevoegen aan het AP Cultureel Erfgoed Event?

**Comment by jitsedc** [comment]
Opnieuw op te nemen bij nieuwe iteratie
